scripts/package-distribution.ps1

```diff
@@ -141,6 +115,17 @@ foreach ($Build in $AllBuilds) {
         $Libraries += Join-Path $ArtifactsDir "msquic.lib"
     }
 
+    # if datapath_raw_xdp_kern.o exists under $ArtifactsDir, $UseXdp to be true
```

Comment:
Yes, this is correct but we shouldn't tie $Time64Distro to this, because for example: Ubuntu 24.04 is still Time64Distro but if we're compiling it against ARM, dependency will get converted to libssl3 from libssl3t64